seismic/network_event_dataset.py

```diff
@@ -229,6 +229,32 @@ def prune(self, items, cull=True):
 
     # end func
 
+    def write(self, output_h5_filename, index_format='event'):
```

Comment:
Convenience function for writing dataset and helps self-document existence of two index formats when writing to hdf5.